README.md

```diff
@@ -1,13 +1,30 @@
 
-Below is a list of information you can use to learn more about us.
+## 📖 Culture Handbook
 
-- [What do we do at EQ Works?](https://docs.google.com/presentation/d/1ymlJorsnuX_n6sDNbBdwu-r5_4ztr2BFI6vs3CT5Bog/present) An overview of our products and industry.
-- [EQWorks Culture](https://github.com/eqworks/common) - an evolving list of  conventions and practices we've adopted to maximize collaboration amongst team members.
-- [EQWorks Work Samples](https://github.com/EQWorks/work-samples) - candidates for our development and data science team are asked to work on a small problems that parallels EQ Works real world challenges. We prefer seeing how you can work on your own to overcome challenges. 
-- [Open-source projects](https://github.com/EQWorks?q=&type=public) - a growing list of open-source projects, made with our humble effort and sincere hope to give back to the community.
-- [LOCUS Engineering](https://medium.com/locus-engineering) - Our development and engineering blog.
+You can find our evolving culture handbook in our [Common project](https://github.com/EQWorks/common). In it, we introduce critical subjects such as:
 
-If you feel like joining our team, check out our [active job postings](https://apply.workable.com/eqworks/).
\ No newline at end of file
+- Our [decision-making](https://github.com/EQWorks/common/blob/main/decision-making.md) principles.
+- **Our [hiring](https://github.com/EQWorks/common/blob/main/hiring.md) philosophy and process**.
+- Our [asynchronous-first communication and collaboration](https://github.com/EQWorks/common/blob/main/communications/README.md) practices.
+- And more!
+
+## 📍 What do we do?
+
+We build a platform of products and features and extend it to an ecosystem to help you understand and leverage geospatial data. You can find [an overview](https://docs.google.com/presentation/d/1ymlJorsnuX_n6sDNbBdwu-r5_4ztr2BFI6vs3CT5Bog/present) of the products and technologies we build and use.
+
+## 💻 Work Samples
+
+At EQ Works, we take a more effective path to the technical interview. Instead of solving abstract algorithms and data structure problems, you'll get to work on an open-book and open-ended work sample that resembles our actual work. If you are a maker, you may find the problems easier (perhaps more enjoyable) yet much more meaningful to understand how well you can integrate into our team from a technical perspective. We encourage you to check out [our hiring philosophy](https://github.com/EQWorks/common/blob/main/hiring.md) to get more insights.
+
+## 🙌 Open Presence
+
+We have a [LOCUS Engineering Blog](https://medium.com/locus-engineering) maintained by our product/engineering teams. In it, we share our thoughts on our product and technology, culture, and more.
+
+You can also find us on GitHub with a growing portfolio of [open-source projects](https://github.com/EQWorks?q=&type=public), made with our humble effort sincere hope to give back to the community.
```

Comment:
humble effort **and** sincere hope?

Reply:
You can find our growing portfolio of [open-source projects](https://github.com/EQWorks?q=&type=public) on GitHub; in the spirit of open contribution to the wider geospatial analytics and machine learning community.

Reply:
good suggestions, will incorp